brave/browser-laptop
# Apply BrowserButton to clipboardButton

- by applying the modified BEM style to dedupe properties
- by replacing span with browserButton
- by setting `clipboard` fa icon by default

Closes #11373

Auditors:

Test Plan:
1. Open about:brave
2. Click the clipboard button
3. Make sure the data is copied

Submitter Checklist:


Test Plan:


Reviewer Checklist:

Tests
